src/rpc/net.cpp

```diff
@@ -299,10 +299,10 @@ UniValue getaddednodeinfo(const UniValue& params, bool fHelp)
 
     std::vector<AddedNodeInfo> vInfo = GetAddedNodeInfo();
 
-    if (params.size() == 2) {
+    if (params.size() == 2 || params.size() == 1 && params[0].type() == UniValue::VSTR) {
```

Comment:
Was first a bit doubtful about this, and going to suggest to only allow skipping the dummy argument when the second argument is not provided. But all in all I like this trick. This allows a future path to getting rid of the argument completely.